dicom_utils/volume.py

```diff
@@ -209,7 +209,7 @@ def handle_dicom(self, dcm: U) -> U:
     def get_indices(self, total_frames: Optional[int]) -> Tuple[int, int, int]:
         if self.center is None and total_frames is None:
             raise ValueError(f"`total_frames` cannot be `None` when `{self.__class__.__name__}.center` is `None`")
-        center = self.center if self.center is not None else total_frames // 2
+        center = self.center if self.center is not None else cast(int, total_frames) // 2
```

Comment:
Would `int(total_frames)` be better?

Reply:
That would still give a type error since `int(None)` isn't defined. The type checker doesn't have a mechanism to understand that the first if statement and the ternary operator will together ensure that `total_frames` is an int once that statement is reached, so I think the best we can do is a cast.